test/local/mailer_tests.js

```diff
@@ -267,6 +267,34 @@ P.all(
               mailer[type](reminderMessage)
             }
           )
+        } else if (type === 'newDeviceLoginEmail') {
+          var location = {
+            city: 'Mountain View',
+            country: 'USA'
+          }
+
+          message = {
+            device: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:48.0) Gecko/20100101 Firefox/48.0',
+            email: 'a@b.com',
+            ip: '219.129.234.194',
+            location: location,
+            timeZone: 'America/Los_Angeles'
+          }
+
+          test(
+            'Location and ip data is present in email template output for ' + type,
+            function (t) {
+              mailer.mailer.sendMail = function (emailConfig) {
+                t.ok(includes(emailConfig.html, location.city + ', ' + location.country))
```

Comment:
Actually, I think it makes more sense to pull these out and iterate over them similar to [this](https://github.com/mozilla/fxa-auth-mailer/blob/3c50620251260592a44e475649b2c76a9f9a7273/test/local/mailer_tests.js#L43) and [this](https://github.com/mozilla/fxa-auth-mailer/blob/3c50620251260592a44e475649b2c76a9f9a7273/test/local/mailer_tests.js#L95). That way if we add more templates that have location, we don't have to duplicate code.